### ISSUE TITLE
Network instructions seem to be outdated

### ISSUE THREAD

**Issue by fvlasie** [redhat-openstack/website#1166]
https://www.rdoproject.org/networking/neutron-with-existing-external-network/

The instructions on this page seem to be out-of-date. I get deprecation warnings and after the section 
8<-----8<----
Now, let's switch to the newly created user:

# export OS_USERNAME=internal
# export OS_TENANT_NAME=internal
# export OS_PASSWORD=foo

Then create a router and set its gateway using the external network created by the admin in one of previous steps:

# neutron router-create router1
8<-----8<----

I get errors and I cannot continue:

[root@soso ~(keystone_admin)]# neutron router-create router1
neutron CLI is deprecated and will be removed in the future. Use openstack CLI instead.
The request you have made requires authentication. (HTTP 401) (Request-ID: req-19e08cea-6fc6-450a-a723-0864ed1cbbb4)

**Comment by karelyatin** [comment]
Yes neutron client is deprecated but it still should work, any way we need to switch to openstackclient which is a recommended client for interacting with OpenStack services why CLI, also _TENANT_ vars deprecated need to use _PROJECT_ now.
